dvc/command/diff.py

```diff
@@ -39,7 +39,7 @@ def _get_md5_string(sign, file_name, checksum):
     def _get_dir_changes(cls, dct):
         engine = inflect.engine()
         changes_msg = (
-            "{} {} not changed, {} {} modified, {} {} added, "
+            "{} {} changed, {} {} modified, {} {} added, "
```

Comment:
Why removing `not`, and if its necessary, what is the difference between changed and modified?

Reply:
1) "Why removing not," --> because "not" isn't needed.  "N files not changed" isn't correct.

2) "what is the difference between changed and modified?" --> I don't know. I asked the same question in the issue. Likely, *changed* encompasses all the files that have been involved in a commit -- new, deleted, edited. Whereas *modified* only those which have existested already in the index and which body was changed.